cardano_node_tests/tests/test_plutus_mint_build.py

```diff
@@ -111,6 +111,10 @@ def test_minting(
         )
         cluster.submit_tx(tx_file=tx_signed_step1, txins=tx_output_step1.txins)
 
+        # check expected fees
+        expected_fee = 168977
+        assert helpers.is_in_interval(tx_output_step1.fee, expected_fee, frac=0.15)
```

Comment:
In my opinion it is best to do these kinds of checks at the very end of the test (right before optional db-sync checks).
In case the expected fee doesn't match the actual fee, we still want to know if the actual functionality works.

So all the checks that are not needed for the "core" functionality (fee checking, `check_tx_view`, etc.) should ideally go to the very end. This will also improve readability of the test as functionality testing and additional checks are not mixed.